src/lang/ts/intermock.ts

```diff
@@ -985,7 +983,8 @@ export function mock(options: Options) {
   if (!fileContents) {
     return {};
   }
-  console.log(fileContents);
+  // remove this from the overall output, maybe wrap in an if with a new arg in options.
+  // console.log(fileContents);
```

Comment:
Let's remove this. No need for commented out console logs :)

Reply:
removed